plugins/tds61_rules.js

```diff
@@ -4491,6 +4481,32 @@ tds61.rules = {
     ],
     // ##### End of shopList #####
 
+    // ##### Start of fuzzyTable #####
+    // This list uses the new IsA, IsSimilar etc functions to map a number of input values to a single output
+    fuzzyTable : [
+     ['ZI016_ROC','7',schemaTools.isA('surface=paved', 0.8, 0.1, 0.5)], // Ridgid Pavement
+     ['ZI016_ROC','3',schemaTools.isA('surface=unpaved', 0.8, 0.1, 0.5)], // Flexible Pavement
```

Comment:
The 0.1 and 0.5 parameters aren't relevant to the isA relationship. This is only relevant with similarTo.